.github/workflows/build_and_deploy_sam_template.yaml

```
@@ -24,6 +24,10 @@ on:
         required: false
         default: uat
         type: string
+      CONFIG_FILE:
```

```suggestion
      TOML_CONFIG_FILE:
```

I think this would be clearer.